qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/componentinstaller/components/FindRouteOnLongPress.kt

```diff
@@ -99,8 +99,8 @@ class FindRouteOnLongPress(
     private suspend fun LocationEngine.getLastLocation() =
         suspendCancellableCoroutine<Location> { cont ->
             getLastLocation(object : LocationEngineCallback<LocationEngineResult> {
-                override fun onSuccess(result: LocationEngineResult) {
-                    result.lastLocation?.also {
+                override fun onSuccess(result: LocationEngineResult?) {
+                    result?.lastLocation?.also {
```

Comment:
coroutine will stuck if `result` will be null, won't it?

Reply:
Good catch!